src/npe2/cli.py

```diff
@@ -105,6 +105,9 @@ def _pprint_table(
             val = ""
             if isinstance(r, dict):
                 val = ", ".join(f"{k} ({v})" for k, v in r.items())
+            # list if overridden by function name
+            elif isinstance(r, type([])):
+                val = ", ".join(str(_) for _ in r)
```

Comment:
```suggestion
            elif isinstance(r, list):
                val = ", ".join(str(text) for text in r)
```

`_` means in python unused variable